CMakeLists.txt

```diff
@@ -273,7 +277,7 @@ install(TARGETS ttnn
 )
 if(WITH_PYTHON_BINDINGS)
     # Install .so into src files for pybinds implementation
-    install(FILES ${PROJECT_BINARY_DIR}/lib/_ttnn.so
+    install(TARGETS ttnn
```

Comment:
Hey same comment as before - how does this SO file get its name properly for the Python bindings?

Reply:
The name of the SO file is defined by a target property.
https://github.com/tenstorrent/tt-metal/blob/e8245e03848ac5a06e0318c276e1387d5bb508a2/ttnn/CMakeLists.txt#L542-L545